app/email.go

```diff
@@ -26,7 +27,10 @@ func SendChangeUsernameEmail(oldUsername, newUsername, email, locale, siteURL st
 	bodyPage.Html["Info"] = template.HTML(T("api.templates.username_change_body.info",
 		map[string]interface{}{"TeamDisplayName": utils.Cfg.TeamSettings.SiteName, "NewUsername": newUsername}))
 
-	if err := utils.SendMail(email, subject, bodyPage.Render()); err != nil {
+	renderedHtmlBody := bodyPage.Render()
```

Comment:
Maybe there should be a function that takes a bodyPage and does all this and send the mail. So this doesn't have to be copy coded everywhere.